src/app/components/Title/title.module.css

```diff
@@ -0,0 +1,9 @@
+.title {
+  text-transform: lowercase;
+  color: white;
+  text-align: center;
+  font-weight: 300;
+  font-family: 'Source Sans Pro', sans-serif;
```

Comment:
You could import the font in global.css and set the font-family for the whole project to 'Source Sans'